documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,294 @@
+
+.Can download `must-gather` logs only using the CLI
+
+The {product-short} web console can no longer download logs. Every `must-gather` log image download must be preformed using CLI commands. See link:https://github.com/openshift/must-gather-operator[Must Gather Operator] for more information.
+
+.Review and configure migrated VMs
+
+The {project-short} web console shows the name and migration status of each VM. The migrated VMs are labeled `app: <vmname>` to help further changes in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.Validation for RHEL 6
+
+Red Hat Enterprise Linux (RHEL) 9 does not support RHEL 6 as a guest operating system, and therefore RHEL 6 is not supported in {ocp-name} Virtualization. A validation of RHEL 6 guest operating system was added. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
+
+.Block migrations from {ocp-name} versions before 4.13
+
+The `Export API` in {ocp-name} Virtualization exposes VM manifests from {ocp-name} 4.13, so source {ocp} clusters must be version 4.13 or later. link:https://issues.redhat.com/browse/MTV-734[(MTV-734)]
+
+.Source and storage endpoints are not selected automatically
+
+Source network and source storage are not selected automatically when creating an {ocp} to {ocp} plan. link:https://issues.redhat.com/browse/MTV-729[MTV729]
+
+.Restored vSphere Verify certificate functionality
+
+The vSphere Verify certificate option in the `add-provider` dialog has been restored. It was removed in the transition to {ocp} console. link:https://issues.redhat.com/browse/MTV-737[(MTV-737)]
+
+.{ocp-name} Console native SDK fetch, post, and delete calls
+
+The `fetch`, `post`, and `delete` calls in the Forklift UI are now native to the {ocp-name} API. link:https://issues.redhat.com/browse/MTV-365[(MTV-365)]
+
+.{project-short} UI plugin uses only a dynamic plugin for the SDK REST API `fetch` methods
+
+The UI plugin uses a dynamic plugin for the SDK REST API when using the `fetch` methods. link:https://issues.redhat.com/browse/MTV-616[(MTV-616)]
+
+.Migration success status bar chart 
+
+A bar chart shows the number of migrations and their status - `starts`, `finished`, and `failed`. link:https://issues.redhat.com/browse/MTV-770[(MTV-770)]
+
+.Improved list of VMs in the detailed view of each provider
+
+The detailed view of each provider includes specific information relevant to the VMs of that provider. link:https://issues.redhat.com/browse/MTV-772[(MTV-772)]
+
+.Edit plans that failed to migrate VMs
+
+Plans that have failed to migrate any VMs can be edited. Some plans fail or are canceled because the network and storage mappings are wrong. These plans can be edited until they succeed. link:https://issues.redhat.com/browse/MTV-779[(MTV-779)]
```

Comment:
```suggestion
.Pans that failed to migrate VMs can now be edited

With this update, you can edit plans that have failed to migrate any virtual machines (VMs). Some plans fail or are canceled because of incorrect network and storage mappings. You can now edit such plans until they succeed. link:https://issues.redhat.com/browse/MTV-779[(MTV-779)]
```

Reply:
changed